src/components/Heading/Heading.js

```diff
@@ -52,6 +64,10 @@ Heading.propTypes = {
    * Optional additional className string to overwrite styles.
    */
   className: PropTypes.string,
+  /**
+   * Adds bottom margin to the heading.
+   */
+  withMargin: PropTypes.bool,
```

Comment:
So, two things:

- If we're going to really make this a prop, can we just call it "margin"? Let's keep the props to one word if possible!
- Otherwise, I think that having a margin it a much more sensible default than not having one.